docs/content/contribution-guidelines/operations/docs-styleguide.md

```diff
@@ -25,21 +25,24 @@ The goal of written documentation should be "just enough." Enough detail that th
 
 It should also be **clear**: terms should be defined and jargon explained when first used. Long, convoluted explanations should be examined to see if they might be better broken up into smaller chunks.
 
-Much of KubeStellar's existing documentation -- especially introductory and overview text -- is written with a slightly light and breezy tone. That is deliberate; it, however, does not mean that the texts should not be technically accurate.
+Much of KubeStellar's existing documentation -- especially introductory and overview text -- is written with a slightly light and breezy tone. That is deliberate it; however, does not mean that the texts should not be technically accurate.

```

Comment:
This change introduces a grammar problem.

> That is deliberate it; however, does not mean ...

Is just wrong in the change in the use of "it".